Cargo.toml

```diff
@@ -598,7 +597,7 @@ move-package = { path = "third_party/move/tools/move-package" }
 move-prover = { path = "third_party/move/move-prover" }
 move-prover-boogie-backend = { path = "third_party/move/move-prover/boogie-backend" }
 move-stackless-bytecode = { path = "third_party/move/move-prover/bytecode" }
-move-stdlib = { path = "third_party/move/move-stdlib" }
+aptos-move-stdlib = { path = "aptos-move/framework/move-stdlib" }
```

Comment:
Interesting. Those use cases where in fact referring to -- let me call it -- the reference implementation of the Move stdlib in the Move repo. Which is not invalid to do. But perhaps cleaner to refer to our own.